include/urdf_utilities/urdf_utilities.h

```diff
@@ -0,0 +1,60 @@
+#ifndef URDF_UTILITIES_H
+#define URDF_UTILITIES_H
```

Comment:
roslint: `URDF_UTILITIES_URDF_UTILITIES_H`

I'd encourage to add a roslint test